CHANGELOG.md

```diff
@@ -43,6 +43,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Copy `build.sh` over from opensearch-build ([#4887](https://github.com/opensearch-project/OpenSearch/pull/4887))
 - Add project health badges to the README.md ([#4843](https://github.com/opensearch-project/OpenSearch/pull/4843))
 - Added changes for graceful node decommission ([#4586](https://github.com/opensearch-project/OpenSearch/pull/4586))
+- Use timeout support of the IndexSearcher instead of custom implementation([#4906](https://github.com/opensearch-project/OpenSearch/pull/4906))
```

Comment:
Probably a "skip-changelog" case now since this is a refactoring and not a functional change, right?

Reply:
Thanks @andrross , have a loooong discussion with Apache Lucene team, will postpone this future a bit (for 2 reasons 1) there are a number of improvements in work 2) the way we use `IndexSearcher` does not allow us to use this feature as-is, without copying code